pom.xml

```diff
@@ -67,7 +67,7 @@
 
     <properties>
         <jahia-depends>jahia-authentication</jahia-depends>
-        <export-package>org.jahia.modules.jahiaoauth.service,org.jahia.modules.jahiaoauth.action,com.github.scribejava.*</export-package>
```

Comment:
question: why did you remove this package export?

Reply:
I don't know why this package was removed. This package is necessary to extend jahia-oauth in a custom module.

Reply:
yes, we cannot remove it